ci_tools/check_slots.py

```diff
@@ -26,82 +28,132 @@
 missing_attribute_nodes = []
 missing_from_all = []
 
+ErrorCollection = {
+    'missing_all':[],
+    'non_alphabetical_all':[],
+    'missing_slots':[],
+    'overridden_slots':[],
+    'missing_attribute_nodes':[],
+    'missing_from_all':[]
+}
+
+def extract_dict_elements(input_dict):
+    """
+    Extracts specific elements from the input dictionary based on the given keys and returns a new dictionary.
+
+    Args:
+        input_dict (dict): The input dictionary from which elements will be extracted.
+
+    Returns:
+        dict: A new dictionary containing the extracted elements.
+    """
+    output_dict = {'title':"Pyccel_lint", 'summary':"Check Slots:\n\n",'annotations':[]}
+    for values in input_dict.items():
+        first_iteration = True
+        for value in values[-1]:
+            if value:
+                if first_iteration:
+                    output_dict['summary'] += f"{value['title']} :\n\n"
+                    first_iteration = False
+                output_dict['summary'] += f"\t-{value['message']}\n\n"
```

Comment:
Avec le syntax markdown recommendé par le linter je crois que ça devrait être:
```suggestion
                output_dict['summary'] += f"-   {value['message']}\n\n"
```